### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/guide/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#617] [Announcements]
# https://plugin-comment2.vuejs.press/guide/

Setting Options You can both set options with plugin options on Node side and set options in client config file on Browser side. tab With Plugin Options tab With Client Config F...

https://plugin-comment2.vuejs.press/guide/

**Comment by realpdai** [comment]
test

**Reply by realpdai** [reply]
test - reply comments

+ could not edit the comment
+ could not delete my own comment

**Comment by dream-oyh** [comment]
test
